rplugin/python3/nvim-typescript/__init__.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, vim):
         self.files = Dir()
         self._last_input_reload = time()
         self.cwd = os.getcwd()
+        self.src = 0
```

Comment:
If you could just rename this to something more explicit like `self.highlight_source` ?